CHANGELOG.md

```diff
@@ -1,4 +1,18 @@
 ## 3.68.0 (Unreleased)
+FEATURES:
+* **New Resource:** pubsub: Added schema resource as `google_pubsub_schema` to google pubsub ([#3243](https://github.com/hashicorp/terraform-provider-google-beta/pull/3243))
+
+IMPROVEMENTS:
+* compute: Fixed permadiff for `cdn_policy.serve_while_stale` and `cdn_policy.*_ttl` in `google_compute_backend_service` (beta) ([#3230](https://github.com/hashicorp/terraform-provider-google-beta/pull/3230))
```

Comment:
Duplicate entries? Also these are bug fixes, I think

Reply:
deleted entry and moved to bug